modules/rosa-sdpolicy-security.adoc

```diff
@@ -30,6 +30,7 @@ In addition to normal users, {product-title} provides access to an {product-titl
 - Can add and manage `NetworkPolicy` objects.
 - Are able to view information about specific nodes and PVs in the cluster, including scheduler information.
 - Can access the reserved `dedicated-admin` project on the cluster, which allows for the creation of service accounts with elevated privileges and also gives the ability to update default limits and quotas for projects on the cluster.
+- Can install Operators from OperatorHub (`\*` verbs in all `*.operators.coreos.com` API groups).
```

Comment:
Sorry, I am not I follow this bit in the parentheses:

> ... (`\*` verbs in all `*.operators.coreos.com` API groups).

Also see IBM Style guidance on [parentheses](https://www.ibm.com/docs/en/ibm-style?topic=punctuation-parentheses#general-text)